lib/encrypt_mails.rb

```diff
@@ -32,15 +32,14 @@ def mail_with_relocation(headers={}, &block)
       act = Setting.plugin_openpgp['activation']
       return mail_without_relocation(headers, &block) if
         act == 'none' or not actions.include? @_action_name or
-        (act == 'project' and not project.try('module_enabled?', 'openpgp'))
+        (act == 'project' and not project.try('module_enabled?', 'openpgp') and not @_action_name == 'lost_password')
```

Comment:
Why is this necessary? I am confused. Shouldn't `actions.include? @_action_name` return True for `lost_password`?